### PR TITLE
Set Metadata when creating new positions in repository migration. 

### PR DESCRIPTION
**Pull request by njohner**
We add support for the metadata fields (`IClassification` and `ILifeCycle` behaviours) for repository folders created during repository migration.

For [CA-3245]

[CA-3245]: https://4teamwork.atlassian.net/browse/CA-3245?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: opengever/maintenance/scripts/repository_migration.py [hunk 1]
```diff
@@ -859,6 +894,35 @@ def extract_permissions(self, row):
 
         return permissions
 
+    def extract_metadata(self, row):
+        metadata = {}
+        for field in metadata_fields:
+            fieldname = field.getName()
+            value = getattr(row, fieldname)
```
Comment (lukasgraf) [inline opengever/maintenance/scripts/repository_migration.py:913]: `getattr()` without an explicit default as the third argument will raise `AttributeError` (depending on the object - not exactly sure what type `row` is here). Just making sure this is deliberate.

File: opengever/maintenance/scripts/repository_migration.py [hunk 2]
```diff
@@ -895,6 +959,9 @@ def insert_label_row(self, sheet):
             'Verletzt Leafnode Prinzip',
             'Ist ungultig',
 
+            # metadata
+            'Metadaten Nichtueberstimmung',
```
Comment (lukasgraf) [inline opengever/maintenance/scripts/repository_migration.py:979]: That doesn't quite work in German 😉  I'd say `Metadaten stimmen nicht ueberein`, unless you need a noun-ified message. Then I'd say `Metadaten-Aenderung`.

**Review by lukasgraf — APPROVED** [review]
LGTM! 👍 
Just some trivialities.

File: opengever/maintenance/scripts/repository_migration.py (hunk 1 above)
Comment (njohner) [reply opengever/maintenance/scripts/repository_migration.py:913]: row has an attribute for every key in `metadata_fields`.

File: opengever/maintenance/scripts/repository_migration.py (hunk 2 above)
Comment (njohner) [reply opengever/maintenance/scripts/repository_migration.py:979]: 😅

**Pull request merged**